### PR TITLE
Adding in values need for edx-rbac/edx-enterprise

### PR DESCRIPTION
**Pull request by christopappas**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

**Comment by feanil** [comment]
@Agrendalath

**Comment by Agrendalath** [comment]
@feanil, I'm afraid I don't have context on this PR.

File: playbooks/roles/edxapp/defaults/main.yml [hunk 1]
```diff
@@ -1240,6 +1243,9 @@ generic_env_config:  &edxapp_generic_env
     JWT_AUTH_COOKIE_SIGNATURE: "{{ EDXAPP_JWT_AUTH_COOKIE_SIGNATURE }}"
     JWT_AUTH_REFRESH_COOKIE: "{{ EDXAPP_JWT_AUTH_REFRESH_COOKIE }}"
 
+  # edx-rbac Setting
+  SYSTEM_WIDE_ROLE_CLASSES: "{{ EDXAPP_SYSTEM_WIDE_ROLE_CLASSES }}"
```
Comment (coryleeio) [inline playbooks/roles/edxapp/defaults/main.yml:1247]: @christopappas I just want to confirm, you do want this in the lms and the cms, correct?

Comment (coryleeio) [reply playbooks/roles/edxapp/defaults/main.yml:1247]: Confirmed this in slack

**Review by coryleeio — APPROVED** [review]

**Pull request merged**